conf/config-default.yaml

```diff
@@ -379,6 +379,7 @@ plugins:                          # plugin list (sorted by priority)
   # <- recommend to use priority (0, 100) for your custom plugins
   - example-plugin                 # priority: 0
   #- skywalking                    # priority: -1100
+  #- opentelemetry                 # priority: -1200
```

Comment:
can we add a plugin_attr there as a reference? Similar to 
https://github.com/apache/apisix/blob/94088cde46a0c12149ba8dcfeb7bf4bfdb1ff869/conf/config-default.yaml#L403-L411